src/lib/ci.js

```diff
@@ -0,0 +1,47 @@
+
+export const CI_HOST = "https://lighthouse-dot-webdotdevsite.appspot.com";
```

Comment:
`LH_HOST`

and maybe we rename this file to lighthouse-service.js or something...?